api-schema/src/main/java/org/stellar/anchor/api/custody/fireblocks/CreateTransactionRequest.java

```diff
@@ -0,0 +1,104 @@
+package org.stellar.anchor.api.custody.fireblocks;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class CreateTransactionRequest {
+  private String assetId;
+  private TransferPeerPath source;
+  private DestinationTransferPeerPath destination;
+  private TransactionRequestDestination[] destinations;
+  private String amount;
+  private Boolean treatAsGrossAmount;
+  private String fee;
+  private String gasPrice;
+  private String gasLimit;
+  private String networkFee;
+  private String priorityFee;
+  private FeeLevel feeLevel;
+  private String maxFee;
+  private Boolean failOnLowFee;
+  private Boolean forceSweep;
+  private String note;
+  @Deprecated private Boolean autoStaking;
```

Comment:
What is the reason to keep the `@Deprecated` fields?

Reply:
At the moment we added those classes we weren't sure which fields we will need to use. That's why we added all listed in https://docs.fireblocks.com/api/?python#create-a-new-transaction

We will remove deprecated fields in a next commit

Reply:
Fixed in https://github.com/stellar/java-stellar-anchor-sdk/pull/1067